socorro/processor/general_transform_rules.py

```diff
@@ -53,7 +53,7 @@ def _action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
         processed_crash.os_name = ''
         processed_crash.os_version = ''
 
-        system_info = processed_crash.json_dump.get('system_info')
+        system_info = processed_crash.get('json_dump').get('system_info')
```

Comment:
Missing the `, {}` default. 
If `processed_crash` is `{'foo': 'bar'}` this line would raise an AttributeError since `None` doesn't have a `get` method.